--- old/qtbase/src/widgets/styles/qstylesheetstyle.cpp
+++ new/qtbase/src/widgets/styles/qstylesheetstyle.cpp
@@ -1465,7 +1465,6 @@ void QRenderRule::configurePalette(QPalette *p, QPalette::ColorGroup cg, const Q
         QBrush placeholder = pal->foreground;
         placeholder.setColor(phColor);
-        p->setBrush(cg, QPalette::PlaceholderText, placeholder);
     }
     if (pal->selectionBackground.style() != Qt::NoBrush)
         p->setBrush(cg, QPalette::Highlight, pal->selectionBackground);
